istioctl/pkg/auth/upgrade.go

```diff
@@ -247,12 +263,12 @@ func (ug *Upgrader) addRoleNameToWorkloadLabelMapping(serviceRolePolicy model.Co
 // mapRoleNameToWorkloadLabels maps the roleName from namespace to workload labels that its rules originally
 // applies to.
 func (ug *Upgrader) mapRoleNameToWorkloadLabels(roleName, namespace, fullServiceName string) error {
-	// TODO(pitlv2109): Handle when services = "*" or wildcards.
-	if strings.Contains(fullServiceName, "*") {
-		return fmt.Errorf("services with wildcard * are not supported")
+	// TODO: Handle suffix for services as described at https://istio.io/docs/reference/config/authorization/istio.rbac.v1alpha1/#AccessRule
+	if fullServiceName == "*" {
```

Comment:
what happens if the service name include `*`?

Reply:
This ServiceRole will apply to all services in this namespace (there won't be a workload selector in AuthorizationPolicy).

Reply:
I mean what happens for something like `*-app.svc.default.cluster.local`, previously it returns an error, right?

Reply:
I see. Let's support it in the next PR to reduce the complexity of this PR.